fsharp-backend/src/LibExecution/StdLib/LibString.fs

```diff
@@ -138,25 +138,23 @@ let fns : List<BuiltInFn> =
       sqlSpec = NotYetImplementedTODO
       previewable = Pure
       deprecated = NotDeprecated }
-    //      { name = fn "String" "toInt" 0
-//      ; parameters = [Param.make "s" TStr]
-//      ; returnType = TInt
-//      ; description = "Returns the int value of the string"
-//      ; fn =
-//         (function
-//         | _, [DStr s] ->
-//             let utf8 = Unicode_string.to_string s in
-//             ( try DInt (Dint.of_string_exn utf8)
-//               with e ->
-//                 Exception.code
-//                   utf8
-//                   "\\d+"
-//                   "Expected a string with only numbers" )
-//         | args ->
-//             incorrectArgs ())
-//      ; sqlSpec = NotYetImplementedTODO
-//      ; previewable = Pure
-//      ; deprecated = ReplacedBy(fn "" "" 0) }
+    { name = fn "String" "toInt" 0
+      parameters = [Param.make "s" TStr ""]
+      returnType = TInt
+      description = "Returns the int value of the string"
+      fn =
+        (function
+        | _, [DStr s] ->
+            ( try
+                bigint (s |> int)
```

Comment:
You can't coerce to ints or you'll lose data. If you add a test case for a really large int (> 64 bits), you'll see.

As it happens, I implemented something similar for another test (just merged to main). You want System.Numerics.BigInteger.Parse.